README.md

```diff
@@ -86,6 +86,8 @@ protoc \
 
 The `generated` folder will now contain both `pb_service.js` and `pb_service.d.ts` files which you can reference in your TypeScript project to make RPCs.
 
+**Note** Both `js` and `d.ts` service files will be generated regardless of whether there are service definitions in the protos to allow Bazel to consume the plugin.
```

Comment:
This comment feels redundant for most use cases. I would prefer a unit test, with a descriptive name, which checks the required files are generated

Reply:
Sgtm